crates/tests/win32/tests/win32.rs

```diff
@@ -126,28 +126,31 @@ fn com() -> windows::core::Result<()> {
     unsafe {
         let stream = CreateStreamOnHGlobal(0, true)?;
         let values = vec![1, 20, 300, 4000];
-        let copied = stream.Write(values.as_ptr() as _, (values.len() * core::mem::size_of::<i32>()) as u32)?;
 
+        let mut copied = 0;
+        let hr = stream.Write(values.as_ptr() as _, (values.len() * core::mem::size_of::<i32>()) as u32, &mut copied);
+        assert!(hr.is_ok());
```

Comment:
Generally if a function previously returned `Result` and now returns `HRESULT` you can just turn the `HRESULT` back into a `Result` using `ok` so `stream.Write(...).ok()?` and the error propagation will work as before.

Reply:
Ah great, will clean that up. Thanks!